packages/closer/components/custom-pages/CustomTextCard.tsx

```
@@ -0,0 +1,29 @@
+    body: string;
+  };
+}> = ({ content, settings }) => (
+  <section
+    className='max-w-2xl px-4 mx-auto flex flex-col gap-4'
+  >
+   
+    <div className="w-full flex flex-col gap-4 ">
+      <Heading level={2} className="text-3xl text-foreground">
+          {content?.title}
+      </Heading>
+      <div className="rich-text" dangerouslySetInnerHTML={{ __html: content.body }} />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)